lib/pbench/test/functional/server/test_put.py

```diff
@@ -334,6 +334,7 @@ def test_list_anonymous(self, server_client: PbenchServerClient):
         we see is public. We do care that we don't get an empty list, since we
         uploaded datasets with access public.
         """
+        server_client.auth_token = None
```

Comment:
This hack bothers me. Fix the fixture to restore the previous function scope login behavior, which was intended to support test cases like this!

Reply:
You fixed the fixture, so this should go ...